### ISSUE TITLE
rgb_motion_1d features

### ISSUE THREAD

**Issue by sgarbanti** [facebookresearch/grounded-video-description#24]
Hi, I can't figure out with this features.
The training, validation and test features, both the visuals "_resnet.npy" and the motions "_bn.npy", are the same used in [densecap](https://github.com/LuoweiZhou/densecap); same values and same checksums.

So I don't understand why the features that you provided with the update of 04/15/2020, for the hidden test-set, are different in shape and values compared to those used in densecap.

It's important for me to understand this because I have extended your model to face the dense video captioning challenge on ActivityNet but I got a very low score compared to what I got for the validation set.

**Comment by LuoweiZhou** [comment]
@sgarbanti Sorry for the inconvenience. Could you be more specific on the differences (e.g., shape and values)? This could speed up debugging from my side. Also, just to confirm, you're participating in Sub-Task II, right?

**Comment by sgarbanti** [comment]
Yes I'm partecipating in Sub-Task II of ActivityNet.
Since I saw that the GVD and densecap features were the same (same checksums and obviously, reading the npy files, same values) I used,  for the challenge, the video features of the densecap testset.
The features that you provided with the last update are referred to 3128 videos in common with the 5044 of the activityNet challenge, but I have seen that these new features differ from those of densecap.

For example "It9tfX8Ux2o_bn.npy" and "It9tfX8Ux2o_resnet.npy" have the shapes (459, 1024) and (459, 2048) respectively, while in densecap they are (344, 1024) and (344, 2048).

So I don't understand why for the training set, validation set and test set of GVD the features are the same as in densecap, while for the hidden test-set they should be different.

**Comment by LuoweiZhou** [comment]
@sgarbanti I see. The IDs in ANet-Entities (GVD) on the hidden-test set have some overlappings with the test set from ANet-Captions (densecap), but are scrambled for anonymity purposes. What you can do is keeping them separately:

i) For ANet-Entities Challenge (on grounding), download the features from [here](https://github.com/facebookresearch/grounded-video-description#data-preparation), which include the *training*, *validation*, *testing*, and *hidden_test* on ANet-Entities.

ii) For ANet-Captions Challenge (on densecap), download the features from [here](https://github.com/salesforce/densecap#annotation-and-feature), which has the entire ANet-Captions, including *training*, *validation*, *testing* from ANet-Entities and the original *testing* from ANet-Captions.

That's a really good question/feedback. Thanks and good luck on the challenge!

**Comment by sgarbanti** [comment]
@LuoweiZhou Ok thank you very much and thank you again for shared your work.
Have a good day!
